tests/test_validation_framwork/test_metric_calculators.py

```diff
@@ -70,8 +69,12 @@ def test_MetadataMetrics_calculator():
     df = make_some_data()
     data = df[['ref', 'k1']]
 
-    metriccalc = MetadataMetrics(other_name='k1')
-    res = metriccalc.calc_metrics(data, gpi_info=(0, 0, 0))
+    with warnings.catch_warnings():
```

Comment:
When test data is generated it is expected that e.g. R cannot be calculated (because there is no noise in the data), therefore we ignore these warnings in the tests